crates/context_aware_config/src/api/dimension/handlers.rs

```diff
@@ -50,6 +54,10 @@ async fn create(
     let create_req = req.into_inner();
     let schema_value = create_req.schema;
 
+    validate_dimension_position(
```

Comment:
we also need to check that the position given is not greater than the already existing number of dimensions